Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -290,7 +284,7 @@ import BraintreeDataCollector
                 let dataCollector = BTDataCollector(apiClient: self.apiClient)
                 self.clientMetadataID = self.payPalRequest?.riskCorrelationId ?? dataCollector.clientMetadataID(pairingID)
                 self.sendAnalyticsEvent(for: request.paymentType, success: error == nil)
-                self.handlePayPalRequest(with: approvalURL, error: nil, paymentType: request.paymentType, completion: completion)
+                self.handlePayPalRequest(with: approvalURL, paymentType: request.paymentType, completion: completion)
```

Comment:
We were always passing the error as nil here. Above, we cast `error` as `NSError` which [always succeeds](https://stackoverflow.com/questions/51602907/why-can-any-error-be-unconditionally-converted-to-nserror). We removed error from this function signature and return the error above (instead of `httpResponseMissingUserInfoJSON`). This matches the [Obj-C implementation more similarly](https://github.com/braintree/braintree_ios/blob/b694fc747142a465831b661c4e41732a73ae23e8/Sources/BraintreePayPal/BTPayPalClient.m#L140-L167) and the error was previously only passed here because the completion could be `nil` and it cannot in Swift. This allowed us to also remove `httpResponseMissingUserInfoJSON` and pass the error directly.